tests/containers/rootless_podman.pm

```diff
@@ -91,7 +94,21 @@ sub run {
     ensure_serialdev_permissions;
     select_console "user-console";
 
-    return if softfail_and_skip_on_bsc1182874();
+    # By default the storage driver is set to btrfs if /var is in btrfs
+    # but if the home partition is not btrfs podman commands will fail with
+    # Error: "/home/bernhard/.local/share/containers/storage/btrfs" is not on a btrfs filesystem
+    if (script_output("podman info 2>&1", proceed_on_failure => 1) =~ m/prerequisites for driver not satisfied/) {
+        record_soft_failure("bsc#1197093 - /home partition is in different filesystem");
+
+        $self->select_serial_terminal;
```

Comment:
A better workaround might be to `mkdir -p ~/.config/containers/ && sed \'s/^driver.*/driver \= \"overlay\"/\' /etc/containers/storage.conf > .config/containers/storage.conf`. That doesn't need root.

Reply:
This way I get a new error:
```
podman info
Error: error creating runtime static files directory: mkdir /var/lib/containers/storage: permission denied
```
http://fromm.arch.suse.de/tests/5980#step/rootless_podman/181